docs/FAQ.md

```diff
@@ -0,0 +1,133 @@
+---
+sidebar_position: 6
+---
+
+# FAQ
+
+## What is double entry accounting?
+
+Double entry accounting tracks transfers between accounts. It stores
+the amount of the transfer along with where the amount came from and
+where it goes.
+
+## Why does double entry accounting matter?
+
+Double entry gives you something better than raw data consistency,
+namely financial consistency, by acting as a financial error
+correcting code to keep the books balanced.
+
+It does this just like Newton’s Third Law—for every movement of money
+into an account, there is an equal and opposite movement of money from
+a different account. This ensures that money cannot be created or
+destroyed, but is merely transferred from one account to another, just
+like the law of conservation of energy.
+
+Because it’s so simple, it’s also highly composable. And so it gives
+you tremendous flexibility to model your business, even as product
+lines come and go, or as your business evolves. We have often seen
+organizations rediscover double entry accounting over time, requiring
+painful database migrations as they evolve toward double entry.
+
+Double entry accounting is the language of business, the world
+over. It’s the tried and tested schema for representing any kind of
+business, any kind of transaction. While you can think of double entry
+as the perfect way to describe any kind of financial state transition,
+it’s also the perfect way to track and “account” for anything, any
+kind of business or real world event, not necessarily financial.
+
+“What advantages does he derive from the system of book-keeping by
+double entry? It is among the finest inventions of the human mind;
+every prudent master of a house should introduce it into his economy.”
+— Goethe, 1795
+
+## Why would I want a dedicated distributed database for accounting?
+
+You can do accounting with any database, it’s true. Existing companies
+use PostgreSQL or MongoDB or any other general-purpose
+database. However, this leaves significant work on the user and
+operator. You must figure out high availability, fault-tolerance, and
+consistency. You must figure out how to scale as you add users and
+transfers. And of course, you must re-implement all the double-entry
+and ledger primitives, which are subtle to get right!
+
+TigerBeetle is focused on just one thing: accounting reliably at
+scale. TigerBeetle does double-entry accounting and as a database has
+only two data types: accounts and transfers of money between
+accounts. TigerBeetle also goes beyond typical ledgers in that it’s
+not only great at tracking money within a system, but has a built-in
+two-phase commit coordinator to help you track money as it moves
```

Comment:
What does two-phase commit mean in this case ?

Reply:
Great question and thanks DJ! I've added a new section "What is two-phase commit" to the FAQ